vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/split_testing_frequency_cap.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_FREQUENCY_CAPPING_EXCLUSION_RULES_SPLIT_TESTING_FREQUENCY_CAP_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_FREQUENCY_CAPPING_EXCLUSION_RULES_SPLIT_TESTING_FREQUENCY_CAP_H_
+
+#include <string>
+
+#include "bat/ads/internal/bundle/creative_ad_info.h"
```

Comment:
not needed as forward declared below `struct CreativeAdInfo`

Reply:
deleted